nixos/modules/services/backup/restic.nix

```diff
@@ -384,10 +384,11 @@ in
       ${lib.optionalString (backup.environmentFile != null) "source ${backup.environmentFile}"}
       # set same environment variables as the systemd service
       ${lib.pipe config.systemd.services."restic-backups-${name}".environment [
-        (lib.filterAttrs (_: v: v != null))
+        (lib.filterAttrs (n: v: v != null && n != "PATH"))
         (lib.mapAttrsToList (n: v: "${n}=${v}"))
         (lib.concatStringsSep "\n")
       ]}
+      PATH=$PATH:${config.systemd.services."restic-backups-${name}".environment.PATH}
```

Comment:
Shouldn't we prepend to prioritize executables from the systems unit? Otherwise LGTM